nixos/modules/services/networking/shout.nix

```diff
@@ -33,14 +48,35 @@ in {
 
     configFile = mkOption {
       type = types.nullOr types.lines;
-      default = null;
+      default = configFile;
       description = ''
         Contents of Shout's <filename>config.js</filename> file. If left empty,
-        Shout will generate from its defaults at first startup.
+        It will generated from the <literal>config</literal> attributes, merged
```

Comment:
This description is a little bit misleading; upon first reading it suggested to me that setting it to null (aka empty)  would generate a config file, but in the implementation  it results in an empty string (meaning `config` is ignored outright).

I wonder, does it ever make sense to actually run with a totally empty config file? If not, perhaps make this `types.lines` and remove the `if isNull` logic in the implementation.

As a non-user I have no idea what makes sense, please ignore if non-sense.